tests/python/relay/aot/riscv.mk

```diff
@@ -0,0 +1,85 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
I tried to upstream a patch like this, before and if was rejected because of the `AOTTestRunner` being removed at some point a favor of the MicroTVM Project API interface. (See https://github.com/apache/tvm/pull/12534#discussion_r961100014)

I actually have MicroTVM Project API template for Spike for a long time now (and use it regulary). I still need to clean it up before upstreaming: https://github.com/PhilippvK/microtvm-spike-template

Reply:
I added Project API template for RISC-V with Spike based on CRT

Reply:
Thank you for coming up with that change. Would it be possible to split this up into a different PR with separate tests etc.?

That would make reviewing the individual changes much easier.

Reply:
Yes, sure. I'll create a new PR Project API